streamly.cabal

```diff
@@ -597,77 +582,96 @@ test-suite pure-streams-streamly
     , hspec             >= 2.0   && < 3
   default-language: Haskell2010
 
+test-suite Prelude
+  import: test-options, prelude-test-options
+  type: exitcode-stdio-1.0
+  main-is: Streamly/Test/Prelude.hs
+  ghc-options: -main-is Streamly.Test.Prelude.main
```

Comment:
Why do you need `-main-is`?

Reply:
`-main-is` seems to be there is all the `test-suite`s

Reply:
It allows me to use a module name other than Main, which makes it easier to compile all the test files with ghc directly.